client/src/App.js

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
-
+import Signup from './components/Signup/Signup.js';
 import Login from './components/Login/Login';
 
 function App() {
   return (
     <div className='App'>
       <Router>
         <Switch>
+          <Route exact path='/' component={Signup} />
```

Comment:
just give the path = /signup  
so that we can include the we can use / route for the home page